docs/standard/library.md

```diff
@@ -26,11 +26,17 @@ The various .NET runtimes implement specific versions of the .NET Standard Libra
 
 ## .NET Platforms Support
 
-You can see the complete set of .NET runtimes that support the .NET Standard Library.
+The following table shows the compatibility matrix of the .NET platforms and the versions of .NET Standard:
```

Comment:
While I like the way you phrased it, I wonder whether *compatibility matrix* makes it sound too complicated. I'm not a doc writer, but every time I hear *matrix* it reminds me of math, exams, and frustration :-)